bleak/backends/bluezdbus/manager.py

```diff
@@ -194,25 +194,51 @@ async def async_init(self):
                 # Add signal listeners
 
                 bus.add_message_handler(self._parse_msg)
+                
+                uuids_list: List[str] = []
+                uuids = params.get('UUIDs')
+                if uuids is not None:
+                    uuids_list = uuids.value
```

Comment:
why `.value`? what is the type of `uuids`?

Reply:
it gets encoded as a dbus variant when the BleakScanner is instantiated

Reply:
Added the type hint

Reply:
perfect :-)